### ISSUE TITLE
lsblk problem with ATAoE drives

### ISSUE THREAD

**Issue by jaymzh** [karelzak/util-linux#233]
AOE devices look like `etherd/e1.1` but lsblk converts this to `etherd!e1.1` and then can't find it in `/dev` and thus can't report on it properly:

```
# lsblk -n -P -o NAME,UUID,LABEL,FSTYPE | grep ether
NAME="etherd!e1.1" UUID="" LABEL="" FSTYPE=""
```

Compare to `blkid`:

```
# blkid | grep ether
/dev/etherd/e1.1: UUID="cb355c94-3bf5-49a2-a76b-f37cee3dca0f" TYPE="xfs"
```

And ftr this is mounted:

```
# df -h /data/fa
/dev/etherd/e1.1  8.8T   33M  8.8T   1% /data/fa
```

**Comment by karelzak** [comment]
Version?

It should be already fixed by commit 92441d5c (v2.27).

**Comment by jaymzh** [comment]
Ah, I should have checked upstream, sorry. Yeah, we're on CentOS6, so it's 2.17 - I'll backport a newer version and confirm. Thanks!!
